unit_test/test_spdm_requester/get_version.c

```diff
@@ -480,13 +480,13 @@ void libspdm_test_requester_get_version_case5(void **state)
     spdm_test_context->case_id = 0x5;
 
     status = libspdm_get_version(spdm_context, NULL, NULL);
-    assert_int_equal(status, RETURN_NO_RESPONSE);
+    assert_int_equal(status, LIBSPDM_STATUS_INVALID_MSG_FIELD);
 }
 
 /**
  * Test 6: on the first try, receiving a Busy ERROR message, and on retry, receiving
```

Comment:
@jyao1 I don't understand how this test works. It says that GET_VERSION is called twice, but it looks like it's only called once.

Reply:
That is because case 5 will return SPDM_ERROR_CODE_BUSY.

Then the GET_VERSION will be sent again.

Reply:
What about [line 141](https://github.com/DMTF/libspdm/blob/1d2f6f30dd585cb39dccbf299db25b10dcb01710/unit_test/test_spdm_requester/get_version.c#L141), that has both the `SPDM_ERROR_CODE_BUSY` and the successful `VERSION` response?

Reply:
Right. Typo, case 6.
It use "static uintn sub_index1 = 0;" to track how many times the interface is called.

If it is first time, the test responder return BUSY. Then the case expect the requester issues GET_VERSION again.
Second time, the test responder return SUCCESS.

Reply:
Right but in `libspdm_test_requester_get_version_case6` `libspdm_get_version` is only called once, not twice. And `libspdm_test_requester_get_version_case6` is only called once according to `cmocka_unit_test(libspdm_test_requester_get_version_case6)` (line 726).

Reply:
Yes, libspdm_get_version() internal implementation will call GET_VERSION twice.

Reply:
Ah I understand now. I had thought that the Responder could not reply to a `GET_VERSION` request with a `Busy` error, but it's only `ResponseNotReady` that it can't reply with.

Reply:
As an aside, I filed https://github.com/DMTF/libspdm/issues/717. Thanks @jyao1